compiler/test/fail_compilation/test17699.d

```diff
@@ -0,0 +1,12 @@
+/*
```

Comment:
Add EXTRA_FILES for documentation purposes (and the gdc dejagnu testsuite)

Reply:
Done